### ISSUE TITLE
InvalidArgumentError 

### ISSUE THREAD

**Issue by isennkubilay** [experiencor/keras-yolo3#225]
hi I got this error. How can I solve this problem? InvalidArgumentError (see above for traceback): logits and labels must have the same first dimension, got logits shape [2028,1] and labels shape [676]
